File: NEsper.Tests/support/eql/SupportPluginAggregationMethodTwo.cs
// ---------------------------------------------------------------------------------- /
// Copyright (C) 2007 Esper Team. All rights reserved.                                /
// http://esper.codehaus.org                                                          /
// ---------------------------------------------------------------------------------- /
// The software in this package is published under the terms of the GPL license       /
// a copy of which has been included with this distribution in the license.txt file.  /
// ---------------------------------------------------------------------------------- /

using System;

using net.esper.eql.agg;

namespace net.esper.support.eql
{
	public class SupportPluginAggregationMethodTwo : AggregationSupport
	{
		public override void Validate(Type childNodeType)
		{
			throw new ArgumentException("Invalid node type: " + childNodeType.FullName);
		}

		public override void Enter(Object value)
		{
			//To change body of implemented methods use File | Settings | File Templates.
		}

		public override void Leave(Object value)
		{
			//To change body of implemented methods use File | Settings | File Templates.
		}

		public override Object Value
		{
			get
			{
				return null;  //To change body of implemented methods use File | Settings | File Templates.
			}
		}

		public override Type ValueType
		{
			get
			{
				return null;  //To change body of implemented methods use File | Settings | File Templates.
			}
		}
	}
} // End of namespace

File: trunk/esper/src/test/java/net/esper/support/eql/SupportPluginAggregationMethodTwo.java
package net.esper.support.eql;

import net.esper.eql.agg.AggregationSupport;

public class SupportPluginAggregationMethodTwo extends AggregationSupport
{
    public void validate(Class childNodeType)
    {
        throw new IllegalArgumentException("Invalid node type: " + childNodeType.getName());
    }

    public void enter(Object value)
    {
        //To change body of implemented methods use File | Settings | File Templates.
    }

    public void leave(Object value)
    {
        //To change body of implemented methods use File | Settings | File Templates.
    }

    public Object getValue()
    {
        return null;  //To change body of implemented methods use File | Settings | File Templates.
    }

    public Class getValueType()
    {
        return null;  //To change body of implemented methods use File | Settings | File Templates.
    }
}
